modules/core/src/main/java/org/apache/ignite/internal/processors/platform/client/cache/ClientCacheSqlQueryRequest.java

```diff
@@ -31,7 +31,7 @@
  * Sql query request.
  */
 @SuppressWarnings("unchecked")
-public class ClientCacheSqlQueryRequest extends ClientCacheDataRequest implements ClientTxAwareRequest {
+public class ClientCacheSqlQueryRequest extends ClientCacheQueryRequest implements ClientTxAwareRequest {
```

Comment:
Looks like the change of super class is unncecessary here.
You don't invoke `updateAffinityMetrics` for this request.

Reply:
This change is only to build correct hierarchy, it's unnecessory now, but this is query request and it's better to be child of common query parent.